apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/InsertTest.cs

```diff
@@ -70,14 +70,31 @@ public void InsertRows()
             Assert.Contains(rowsAfter, r => (string)r["player"] == "Lisa");
         }
 
-        [Fact]
-        public void InsertRow_BadData()
+        public static IEnumerable<object[]> BadDataThrowsOptions
+        {
+            get
+            {
+                yield return new object[] { null };
+                yield return new object[] { new InsertOptions() };
+                yield return new object[] { new InsertOptions() { SkipInvalidRows = false } };
```

Comment:
Nit: remove () from the object initializers from these lines.
(At least, I tend to think they're a bit ugly. But if you actively prefer them, that's fine. :)